src/Control/Director.php

```diff
@@ -390,6 +390,7 @@ public function handleRequest(HTTPRequest $request)
      */
     public static function isManifestFlushed()
     {
+        Deprecation::notice('4.12.0', 'Kernel::isFlushed to be used instead');
```

Comment:
We should standardise the language we use as part of this card.
```suggestion
        Deprecation::notice('4.12.0', 'Use Kernel::isFlushed instead');
```
Also update the phpdoc to match this wording and to use the correct version number.

Reply:
Updated